### DISCUSSION TITLE
Allure report print a lot of logs

### DISCUSSION THREAD

**Discussion by IdanZafrani** [spockframework/spock#1553] [Q&A]
Hey,
I'm using an external report that takes all the logs when I execute any test.
How can I control the logs that print on my report?
I saw at "allure-spock2-2.20.0.jar" a "log message" method that prints all logs from the terminal to my report, and it prints everything and not what I really need.
Thar's a way another way to control the logs?

**Comment by Guter1988** [comment]
Hi @leonard84 
I have the same issue as @IdanZafrani 
Cannot find how can I change the log formatting and level.

Thanks ahead.
Guy

**Reply by rtretyak** [reply]
I don't think 'spock' project has anything to do with your logs or allure reports. Even if somebody would be willing to help you with whatever non-spock problem you have, the amount of details in the original post is just not enough to even understand the problem. At least for me. A sample project or snippet that shows your problem would be a good start.
Typically, the log level and format are controlled by a logging framework config (logback/log4j2 etc.)

**Reply by Guter1988** [reply]
I think you may be missing the question. 

Strate and simple question: how can Spock user configure (it looks like Spock has auto-configuration to stdout log) Logger?

When we apply our Log4J2, it is not affected logs from the Spock run (direct example: when we use multi threads/parallel test, it shows tons of Fork logs).

We may missed something, this why asking.

This is better/clear question @rtretyak ?

**Reply by Vampire** [reply]
Spock itself does not use or configure any logging.
Spock actually has three log statements that are unlikely warnings and are printed using `System.out.printf` currently.

Some other parts like Groovy or JUnit Platform might log stuff, but also those should not configure any logging.
And also there is no transitive dependency on any logging API, so if other parts use logging, they probably use `java.util.logging` which by default indeed logs to the console iirc.

So I agree with @rtretyak, this is not really a Spock question as Spock does not configure any logging for you or even does relevant logging.
As there is no [MCVE](https://stackoverflow.com/help/mcve) showing what is the problem and not even an example of concrete messages that are "the problem", there is not much to help here.
